### ISSUE TITLE
[$10]{MSFT-104} Challenges page: Screen reader doesn't announce the type of element for the 'save filter' and 'clear filter' controls.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2808]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure the screen reader is on. 3. Activate the filter. 4. Use the down arrow to understand the layout of the filter. 5. Notice that the screen reader announces 'save filter' and 'clear filter' instead of 'save filter button' and 'clear filter button'.

**Expected Result**
Screen reader should announce these controls as 'save filter button' and 'clear filter button'.

**Actual Result**
Screen reader announces 'save filter' and 'clear filter' instead of 'save filter button' and 'clear filter button'.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 4. Robust
**WCAG Level**: A
**WCAG Success Criterion**: 4.1.2 Name, Role, Value
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2010

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by Oanh-and-only-Oanh** [comment]
@r0hit-gupta , please make sure you're finishing the other tickets before working on others :)

**Comment by r0hit-gupta** [comment]
@Oanh-and-only-Oanh yes. I am finishing every ticket and then moving on to the next. :)

**Comment by r0hit-gupta** [comment]
@Oanh-and-only-Oanh this was by mistake. Please add open for pickup label.

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by r0hit-gupta** [comment]
PR at https://github.com/topcoder-platform/topcoder-react-utils/pull/37

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2808 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2808 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097497 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097497 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097497 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Closing Ticket due to Priority Shift.  May Revisit at another time.

Note: Ticket dependent on React Utils.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097497<br/><br/>```This is an automated message for crazyk via Topcoder X```
